src/systemctl/systemctl.c

```diff
@@ -3083,24 +3083,30 @@ static int start_unit(int argc, char *argv[], void *userdata) {
         polkit_agent_open_if_enabled();
 
         if (arg_action == ACTION_SYSTEMCTL) {
-                enum action action;
+                enum action action = verb_to_action(argv[0]);
```

Comment:
nitpick: we usually avoid mixing variable declarations and function invocations in one line. See CODING_STYLE